### PR TITLE
OSDOCS-11946: adds MicroShift 4.18 relnotes file to branch

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.18

Issue:
[OSDOCS-11946](https://issues.redhat.com/browse/OSDOCS-11946)

Link to docs preview:
[4-18-0 release notes](https://82971--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-18-release-notes)

QE review:
- N/A internal doc work

Additional information:
[build file PR](https://github.com/openshift/openshift-docs/pull/82972)

### CODE REVIEW

File: microshift_release_notes/microshift-4-18-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+//TODO verify info prior to merge
+[id="microshift-4-18-0-dp_{context}"]
+=== RHEA-2025:XXXX - {microshift-short} 4.18.0 bug fix and security update advisory
+
+Issued: DD Month 2025
+
+{product-title} release 4.18.0 is now available. Bug fixes and enhancements are listed in the link:https://access.redhat.com/errata/RHEA-2025:XXXX[RHEA-2025:XXXX] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHEA-2025:XXXX[RHEA-2025:XXXX] advisory.
+
+See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
```
Comment (GroceryBoyJr) [inline microshift_release_notes/microshift-4-18-release-notes.adoc:130]: This link worked beautifully.

**Review by GroceryBoyJr — COMMENTED** [review]
A few small nits.

File: microshift_release_notes/microshift-4-18-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+{microshift-short} is designed to make control plane restarts economical and be lifecycle-managed as a single unit by the operating system. Updates, roll-backs, and configuration changes consist of simply staging another version in parallel and then - without relying on a network - flipping to and from that version and restarting.
+
+//TODO verify K8s version and another other relevant notes
+[id="microshift-4-18-about-this-release_{context}"]
+== About this release
+Version {product-version} of {product-title} includes new features and enhancements. Update to the latest version of {microshift-short} to receive all of the latest features, bug fixes, and security updates. {microshift-short} is derived from {OCP} {OCP-version} and uses the CRI-O container runtime. New features, changes, and known issues that pertain to {microshift-short} are included in this topic.
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-18-release-notes.adoc:16]: That is expected behavior AFIAK.

File: microshift_release_notes/microshift-4-18-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+//[id="microshift-4-18-known-issues_{context}"]
+//== Known issues
+
+//[id="microshift-4-18-pods-writing-files-excess-memory-limits-crash_{context}"]
+//=== tbd known issues
+
+[id="microshift-4-18-asynchronous-errata-updates_{context}"]
+== Asynchronous errata updates
+
+Security, bug fix, and enhancement updates for {microshift-short} {product-version} are released as asynchronous errata through the Red Hat Network. All {microshift-short} {product-version} errata are https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. For more information about asynchronous errata, read the https://access.redhat.com/product-life-cycles?product=Red%20Hat%20build%20of%20Microshift,Red%20Hat%20Device%20Edge[{microshift-short} Life Cycle].
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-18-release-notes.adoc:111]: Yeah, that's boilerplate. I think you really have to just scroll for one if it isn't a specific z-stream listed in the async notes. There aren't any headings on the list.

File: microshift_release_notes/microshift-4-18-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+Red Hat Customer Portal user accounts must have systems registered and consuming {microshift-short} entitlements for {microshift-short} errata notification emails to generate.
+====
+
+This section is updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {microshift-short} {product-version}. Versioned asynchronous releases, for example with the form {microshift-short} {product-version}.z, are detailed in the following subsections.
+
+//TODO verify info prior to merge
+[id="microshift-4-18-0-dp_{context}"]
+=== RHEA-2025:XXXX - {microshift-short} 4.18.0 bug fix and security update advisory
+
+Issued: DD Month 2025
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-18-release-notes.adoc:126]: Yes

**Pull request merged**